aspnetcore/blazor/components/index.md

```diff
@@ -632,13 +638,17 @@ Consider the following `Expander` component that:
 The `Expander` component is added to a parent component that may call <xref:Microsoft.AspNetCore.Components.ComponentBase.StateHasChanged%2A>:
 
 ```razor
+@page "/expander"
+
 <Expander Expanded="true">
-    <h1>Hello, world!</h1>
+    Expander 1 content
 </Expander>
 
-<Expander Expanded="true" />
+<Expander Expanded="true">
+    Expander 2 content
```

Comment:
Adding child-content here changes the example a bit. The explanation around this section is that having child-content means the parameters get reset:

> The second `Expander` component's `Expanded` value isn't reset because no child content is rendered in the second component.

Reply:
Oh ... yeah ... I was distracted by the other updates. I'll patch that in a sec. Thanks for checking. I would've waited for you to review, but I'm still 🏃😅 trying to get caught up.

Reply:
btw ---- This is OK to catch this now. It's great when these are caught **_BEFORE_** we merge to live.

Reply:
Yeah, no problem. I wouldn't expect you to wait for me to review things, but I'll always take a look when I can.

Reply:
In theory, almost **_everything_** should be reviewed, but when we're this far behind on basic coverage, it's hard to do it and get caught up efficiently. My PRs rapidly step on each other :boom:.

I'm **_still_** planning a UE pass on all of the Blazor topics, and I'm sure that I'll catch little 🐞 that have crept into the docs. I think it's ok until then to have anyone looking at these catch and report the bugs. The community has been very helpful.

I'm glad that you spotted this one before it went live. I'm sure that you've gained great favor with the :pray: *server gods* :pray: around here.

Reply:
Understood. I'm going through the docs on a topic-by-topic basis, just trying to learn all about Blazor as it's something I've not really looked at yet. I'll raise anything I find as I go through. If I ever raise something and you'd like me to fix it up with a PR, just say and I'll put something in.